app/backend/src/couchers/interceptors.py

```diff
@@ -170,7 +170,7 @@ def _sanitize_message(message: Message) -> None:
                 submessage = getattr(message, name)
                 if not submessage:
                     continue
-                if descriptor.label == descriptor.LABEL_REPEATED:
+                if descriptor.is_repeated:
```

Comment:
Is this currently tested? Please confirm or add a test to verify this works. We have had issues with this code section in the past, just want to double check.